about/policy/metadata/index.md

```diff
@@ -6,55 +6,11 @@ tags:
   - metadata
   - sgid
 date: 2020-07-09 09:33:34 -0600
-title: Basic Metadata Guidelines
+title: Basic Metadata Requirements
 categories: []
 ---
 
-By most accounts, creating metadata is right up there with organizing your garage. It can be a chore, but it doesn't have to be.
 
-At AGRC, we have decided to make the process more conversational. In other words, instead of looking at a long list of metadata requirements, we thought it would be easier to talk about the data. This is our solution to metadata procrastination.
-
-We have distilled the basic elements of metadata into a few approachable questions. If you can answer these simple questions about your data, then you're officially on your way to good metadata.
-
-#### Can you answer these questions about your data?
-{: .text-left}
-
-##### Summarize your Data
-{: .text-left}
-
-- What is this dataset?
-- What is the purpose of this dataset?
-
-##### Describe your Data
-{: .text-left}
-
-- What does this dataset represent?
-  - ex: road centerlines represent the centerline of a physical road
-- How was this dataset created?
-  - ex: digitized, GPS, etc.
-- How reliable and accurate is the dataset?
-  - ex: survey-grade, mapping-grade, etc.
-- What is the general update schedule for this dataset?
-  - ex: monthly, yearly, as needed, etc.
-- Are there any warnings or limitations on the data?
-  - ex: don’t use parcels as a legal boundary depiction
-  - ex: don’t use parcels as a complete dataset for geocoding
-  - ex: don’t use zip codes for demographic generalization
-- Are there any common misconceptions or assumptions about this data that should be further explained?
-  - ex: highway milepost markers are not an accurate length measurement. If a highway is lengthened or shortened, the markers are not adjusted.
```

Comment:
**thought**: this seems like good information. maybe it got moved or just removed?